docs/components/stepper/index.html

```diff
@@ -67,7 +65,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Issue Details</span>
```

Comment:
Please update all of these `beta` classes.

Reply:
Changes have been done, please review.